src/share/classes/com/sun/javatest/regtest/config/TestProperties.java

```diff
@@ -270,6 +276,9 @@ class Entry {
                     // add the maxOutputSize for result content
                     maxOutputSize = getInt("maxOutputSize", -1);
 
+                    // add the default test timeout value in seconds
+                    defaultTimeout = Duration.ofSeconds(getInt("timeout.default.seconds", 120));
```

Comment:
Hello Christian, the `TEST.ROOT` (and `TEST.properties`) appear to follow the camel case style property names for most of the configurations https://openjdk.org/jtreg/tag-spec.html#TEST_ROOT. For example, `defaultExecMode`, `maxOutputSize` and so on. To be consistent with that, maybe we should expect this to be `defaultTimeoutSeconds` instead of `timeout.default.seconds`?

Reply:
It's quite a mixture already. There's also `lib.build` and `external.lib.roots` as counter examples.

The idea was to synchronize the key name with the system property exposed to tests in `RegressionScript`.

Reply:
I don't have a strong opinion about this naming in the `TEST.ROOT`/`TEST.properties`, because like you state it's already a mix of naming styles.